tasks/utilities/file_handling/task_transfer_files.wdl

```diff
@@ -6,20 +6,20 @@ task transfer_files {
     String target_bucket
     Int cpu = 4
     Int memory = 8
-    String docker_image = "us-docker.pkg.dev/general-theiagen/theiagen/utility:1.1"
+    String docker_image = "gcr.io/google.com/cloudsdktool/google-cloud-cli:427.0.0-alpine"
     Int disk_size = 100
   }
   meta {
     # added so that call caching is always turned off
     volatile: true
   }
   command <<<
-    file_path_array="~{sep=' ' files_to_transfer}"
+    file_path_array="~{sep=' ' files_to_transfer}" 
 
-    gsutil -m cp -n ${file_path_array[@]} ~{target_bucket}
+    gcloud storage cp -n ${file_path_array[@]} ~{target_bucket}
```

Comment:
gcloud is more efficient with data transfer, I believe. Is this the only reason to change to gcloud, or is there another use case?

Reply:
This was an previous task that I use in the workflow but I can update this task as part of this PR as well

Reply:
If there's an argument to change from gsutil to gcloud, I'm happy to discuss. Given the task_transfer_files is used by quite a few workflows, it would be better to avoid touching it unless we need to to minimize scope of potential impact.

Reply:
I'm for switching to gcloud storage. It's more efficient, and gsutil is a deprecated tool that Google no longer recommends

Reply:
Wow, I need to get with the times. Deprecated, it is; let's move to gcloud!